docs/source/demos/fraud.ipynb

```diff
@@ -267,12 +269,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When we optimize for AUC, we can see that the AUC score from this pipeline is better than the AUC score from the pipeline optimized for fraud cost. However, the losses due to fraud are over 3% of the total transaction amount when optimized for AUC and under 1% when optimized for fraud cost. As a result, we lose more than 2% of the total transaction amount by not optimizing for fraud cost specifically.\n",
+    "When we optimize for AUC, we can see that the AUC score from this pipeline performs better compared to the AUC score from the pipeline optimized for fraud cost; however, the losses due to fraud are over 3% of the total transaction amount when optimized for AUC and around 0.7% when optimized for fraud cost. As a result, we lose more than 2% of the total transaction amount by not optimizing for fraud cost specifically.\n",
     "\n",
```

Comment:
Unless we have a better way of programmatically getting the numbers into the notebook, I think we should probably stick to less quantitative terms in describing the differences.  That'll hopefully future proof us against this type of issue a bit.